frontend/src/app/workflows/model/helper/workflows-hint.ts

```diff
@@ -0,0 +1,34 @@
+import * as CodeMirror from 'codemirror';
+
+export function cmWorkflowsHint(cm) {
+  const noteKeywords = ['workflow', 'externalSystem', 'service', 'command', 'event', 'policy', 'user', 'data', 'page', 'problem'];
+  const pageKeywords = ['pageName', 'text', 'input', 'password', 'button', 'value', 'targetPage'];
```

Comment:
```suggestion
const noteKeywords = ['workflow', 'externalSystem', 'service', 'command', 'event', 'policy', 'user', 'data', 'page', 'problem'];
const pageKeywords = ['pageName', 'text', 'input', 'password', 'button', 'value', 'targetPage'];
const allKeywords = [...noteKeywords, ...pageKeywords];

export function cmWorkflowsHint(cm) {
```

Also I would put each keyword on a new line similar to https://github.com/fujaba/fulib.org/blob/master/frontend/src/modes/symbols.ts

Reply:
Did that and also moved the keywords into its own file.